posthuman/chains.json

```diff
@@ -151,8 +151,8 @@
       "address": "starsvaloper14ftwwcfvhnp3qvxawesjan5duh8rmrkrzdvquy",
       "restake": {
         "address": "stars1k9888fzemeujc7j5t6jgesfhtm6cdg90h3vq8y",
-        "run_time": "every 1 hour",
-        "minimum_reward": 1000000
+        "run_time": "every 1 day",
```

Comment:
This should be a specific time in UTC, but it'll be treated as midnight in this format.